css-fonts-5/Overview.bs

```diff
@@ -220,14 +243,16 @@ given percentage multiplied by the used font size. Negative values are invalid a
   ascent-override: 125%;
   descent-override: 25%;
   line-gap-override: 0%;
+  advance-override: 109%;
 }
 
 &lt;div style="font-family: cool-web-font, fallback-to-local"&gt;Title goes here&lt;/div&gt;
 &lt;img src="https://example.com/largeimage" alt="A large image that you don't want to shift"&gt;
 </pre>
 
-	The image will not be vertically shifted when the user agent finishes loading and
-	switches to use the web font.
+	The image will not be shifted as much when the user agent finishes loading and
```

Comment:
Yes, better to under- than over-promise here

Reply:
👍